types/dicer/index.d.ts

```diff
@@ -0,0 +1,167 @@
+// Type definitions for dicer 0.2
+// Project: https://github.com/mscdex/dicer
+// Definitions by: Robin Goupil <https://github.com/GoupilRobin>
+//                 BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+/// <reference types="node" />
+
+export = Dicer;
+
+import stream = require("stream");
+
+// tslint:disable:unified-signatures
```

Comment:
I see your point, but it's actually faster at compile time to unify the signatures that you can rather than expressing all of these as separate overloads.

Reply:
So I should unify them nevertheless? I mean readability is usually more important than optimization of compile times...

Reply:
On second thought, I misread and thought this would collapse more than it will. Never mind then.